src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html

```diff
@@ -89,19 +89,19 @@ <h2 *ngIf="loggedIn" id="wi-detail-title-click" class="detail-title truncate">{{
       </div>
       <div class="col-md-12 col-sm-12 col-xs-12 detail-assignee-wrap">
           <span
-            *ngIf="!workItem.relationalData.assignees?.length"
+            *ngIf="addNewWI || !workItem.relationalData.assignees?.length"
             (click)="activeSearchAssignee()" 
             class="pull-left fa fa-user-plus user-assign-icon">
           </span>
           <img 
-            *ngIf="workItem.relationalData.assignees?.length"
+            *ngIf="!addNewWI && workItem.relationalData.assignees?.length"
```

Comment:
Same logic as I explained up there. I don't think addNewWI flag is needed anywhere.